integration/test/harvest.yml

```diff
@@ -0,0 +1,36 @@
+Exporters:
+  prometheus1:
+    exporter: Prometheus
+    addr: 0.0.0.0
```

Comment:
`addr` is deprecated as [parameter](https://github.com/NetApp/harvest/tree/main/cmd/exporters/prometheus#parameters), maybe use `local_http_addr` instead